test/mobile/model_test/math_ops.py

```diff
@@ -14,7 +14,6 @@ def pointwise_ops(self):
         b = torch.randn(4)
         t = torch.tensor([-1, -2, 3], dtype=torch.int8)
         r = torch.tensor([0, 1, 10, 0], dtype=torch.int8)
-        t = torch.tensor([-1, -2, 3], dtype=torch.int8)
```

Comment:
?

Reply:
I found that lines 15 and 17 in the file test/mobile/model_test/math_ops.py are duplicated, so I removed them as a convenience. If you think this needs to be discussed in a new issue or if this approach is something you do not wish to see, I sincerely apologize, and I will restore the content of this line in the PR. Additionally, I have a more urgent matter that I hope you can answer for me, as this has been troubling me for quite some time. I am unsure where to add tests for the clamp function for boolean tensors.

Reply:
cc @ezyang